db/schema.rb

```diff
@@ -37,6 +37,17 @@
   add_index "roles", ["name", "resource_type", "resource_id"], name: "index_roles_on_name_and_resource_type_and_resource_id"
   add_index "roles", ["name"], name: "index_roles_on_name"
 
+  create_table "user_movies", force: :cascade do |t|
+    t.integer  "user_id",                      null: false
+    t.integer  "movie_id",                     null: false
+    t.boolean  "on_watchlist", default: false
+    t.datetime "created_at",                   null: false
+    t.datetime "updated_at",                   null: false
```

Comment:
`created_at` and `updated_at` should have defaults of `Time.now` or whatever the equivalent in this language is